rfs/root.go

```diff
@@ -68,13 +70,13 @@ func (r *rfsRoot) Create(ctx context.Context, name string, flags uint32, mode ui
 
 	st := syscall.Stat_t{}
 	if err = syscall.Fstat(fd, &st); err != nil {
-		syscall.Close(fd)
+		_ = syscall.Close(fd)
 		return nil, nil, 0, fs.ToErrno(err)
 	}
 
 	node := r.RootData.NewNode(r.RootData, r.EmbeddedInode(), name, &st)
 	ch := r.NewInode(ctx, node, r.idFromStat(&st))
-	lf := NewRfsFile(fd, name, r.host, r.mirror)
+	lf := NewRfsFile(fd, r.relativePath(name), r.host, r.mirror)
```

Comment:
This is a good catch too... I got so caught up that I forgot to test creating files in subdirectories...